tests/eko/scale_variations/test_expanded.py

```diff
@@ -114,49 +114,50 @@ def scheme_diff(g, k, pto, is_singlet):
         the accuracy of singlet quantities is slightly worse.
         """
         if pto[0] >= 2:
-            diff = g[0] * k * a0 - 2 * a1 * k * g[0]
-        if pto[0] >= 3:
-            b0 = beta_qcd_as2(nf)
-            g02 = g[0] @ g[0] if is_singlet else g[0] ** 2
-            diff += (
-                -2 * a1**2 * g[1] * k
-                + a0**2 * g[1] * k
-                + a1**2 * b0 * g[0] * k**2
-                - 0.5 * a0**2 * b0 * g[0] * k**2
-                - a1 * a0 * g02 * k**2
-                + 0.5 * a0**2 * g02 * k**2
-                + a1**2 * g02 * k**2
-            )
-        if pto[0] >= 4:
-            b1 = beta_qcd_as3(nf)
-            g0g1 = g[0] @ g[1] if is_singlet else g[0] * g[1]
-            g03 = g02 @ g[0] if is_singlet else g02 * g[0]
-            diff += (
-                a0**3 * g[2] * k
-                - 2 * a1**3 * g[2] * k
-                - 1 / 2 * a0**3 * b1 * g[0] * k**2
-                + a1**3 * b1 * g[0] * k**2
-                - a0**3 * b0 * g[1] * k**2
-                + 2 * a1**3 * b0 * g[1] * k**2
-                + a0**3 * g0g1 * k**2
-                - a0**2 * a1 * g0g1 * k**2
-                - a0 * a1**2 * g0g1 * k**2
-                + 2 * a1**3 * g0g1 * k**2
-                + 1 / 3 * a0**3 * b0**2 * g[0] * k**3
-                - 2 / 3 * a1**3 * b0**2 * g[0] * k**3
-                - 1 / 2 * a0**3 * b0 * g02 * k**3
-                + 1 / 2 * a0**2 * a1 * b0 * g02 * k**3
-                + 1 / 2 * a0 * a1**2 * b0 * g02 * k**3
-                - a1**3 * b0 * g02 * k**3
-                + 1 / 6 * a0**3 * g03 * k**3
-                - 1 / 2 * a0**2 * a1 * g03 * k**3
-                + 1 / 2 * a0 * a1**2 * g03 * k**3
-                - 1 / 3 * a1**3 * g03 * k**3
-            )
+            diff = -g[0] * k * a0  # - 2 * a1 * k * g[0]
+        # if pto[0] >= 3:
+        #     b0 = beta_qcd_as2(nf)
+        #     g02 = g[0] @ g[0] if is_singlet else g[0] ** 2
```

Comment:
Why did you comment this instead of just removing (for instance)?

Reply:
I was too lazy to re-derive the higher order expression :innocent: so I left it to remind me/someone - one should do it - not that the expression changed with respect to what @giacomomagni derived ... should I remove it?

Reply:
I would not merge a comment but since @giacomomagni did maybe it should be up to him

Reply:
I'm still not a big fan of this particular test and I would argue we should drop it and do #201 instead